skywalking-sniffer/skywalking-api/src/main/java/com/a/eye/skywalking/plugin/interceptor/enhance/ClassEnhancePluginDefine.java

```diff
@@ -70,8 +69,15 @@
             for (ConstructorInterceptPoint constructorInterceptPoint : constructorInterceptPoints) {
```

Comment:
@ascrutae , before iterator, set the DefaultInterceptor on `any()` matcher, to ensure, EnhancedClassInstanceContext attr initialize always。